dv/uvm/core_ibex/directed_tests/READ.md

```diff
@@ -0,0 +1,18 @@
+# Directed Tests
```

Comment:
Why did you remove the README in the next commit? If you don't want to include it as part of the PR please just drop the commits entirely.

Reply:
Oh and please rename to README.md if you do keep it.

Reply:
There was a documentation check failing so removed it temporarily. Now added, probably it was failing due to the naming typo.